pkgs/development/compilers/mono/default.nix

```diff
@@ -1,73 +1,203 @@
-{ stdenv, fetchurl, bison, pkgconfig, glib, gettext, perl, libgdiplus, libX11, callPackage, ncurses, zlib, withLLVM ? false, cacert }:
+{ stdenv
```

Comment:
Please don't put a single package per line. Put 5-6 at least, it's annoying to have such a long preamble.